lib/seattleflu/db/cli/command/etl/consensus_genome.py

```diff
@@ -317,15 +318,20 @@ def upsert_genome(db: DatabaseSession, sequence_read_set: NamedTuple,
     return genome
 
 
-def upsert_genomic_sequence(db: DatabaseSession, genome: NamedTuple, masked_consensus: dict) -> Any:
+def upsert_genomic_sequence(db: DatabaseSession,
+                            sequence_read_set: NamedTuple,
+                            genome: NamedTuple,
+                            masked_consensus: dict) -> Any:
     """
-    Upsert genomic sequence given a *genome* record and some information from a
-    given *masked_consensus*.
+    Upsert genomic sequence given a *sequence_read_set* record, *genome* record
+    and some information from a given *masked_consensus*.
     """
-    LOG.info(f"Upserting genomic sequence «{masked_consensus['sequence_identifier']}»")
+    sequence_identifier = "".join(
+        [masked_consensus['sequence_identifier'], "-", str(sequence_read_set.id)])
```

Comment:
I noticed that since we're only using the sequence read set id, we could read this from the genome record as `genome.sequence_read_set_id` and avoid passing in `sequence_read_set` to this function.